lib/ExpensiMark.js

```diff
@@ -503,6 +535,32 @@ export default class ExpensiMark {
                     return `!(${g2})`;
                 },
             },
+
+            {
+                name: 'video',
+                regex: /<video[^><]*data-expensify-source\s*=\s*(['"])(\S*?)\1(.*?)>([^><]*)<\/video>*(?![^<][\s\S]*?(<\/pre>|<\/code>))/gi,
+                /**
+                 * @param {string} match The full match
+                 * @param {string} g1 {string} The first capture group
```

Comment:
I like the comments, thank you! Something else I think we should consider is using named capture groups. I think that would really help clarify the variables.